packages/material-ui-unstyled/src/SliderUnstyled/SliderValueLabelUnstyled.js

```diff
@@ -30,12 +31,21 @@ function SliderValueLabelUnstyled(props) {
     {
       className: clsx(children.props.className),
     },
-    <Root className={clsx(classes.offset, className)} theme={theme}>
+    <Root className={clsx(classes.offset, className)} theme={theme} aria-hidden>
+      {children.props.children}
       <span className={classes.circle}>
         <span className={classes.label}>{value}</span>
       </span>
     </Root>,
   );
 }
 
+SliderValueLabelUnstyled.propTypes = {
+  children: PropTypes.element.isRequired,
+  className: PropTypes.string,
+  components: PropTypes.shape({ Root: PropTypes.elementType }),
+  theme: PropTypes.any,
+  value: PropTypes.node,
+};
```

Comment:
`react/prop-types` does not recognize `SliderValueLabelUnstyled` as a component so this might go stale. I'm working on a fix upstream though (camelcase function + proptypes = react component).

The main drive behind this is `children: PropTypes.element.isRequired`. Without it you don't get a helpful error message if you remove the children while debugging.